src/CloudFlareStateInterface.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\cloudflare\CloudFlareStateInterface.
+ */
+
+namespace Drupal\cloudflare;
+
+/**
+ * Tracks rate limits associated with CloudFlare Api.
+ */
+interface CloudFlareStateInterface {
+  /**
+   * Increment the count of tag purges done today.
+   *
+   * @see https://support.cloudflare.com/hc/en-us/articles/206596608-How-to-Purge-Cache-Using-Cache-Tags
+   */
+  public function incrementTagPurgeDailyCount();
```

Comment:
The immediate thing that comes to my mind: **local timezone versus CF's timezone**.. haven't read the article though.